src/utils/image.ts

```diff
@@ -39,3 +46,89 @@ export function resizeImage(file: File, max = 3000): Promise<any> {
     );
   });
 }
+
+export async function resizePredictImage(file: File): Promise<any> {
```

Comment:
avoid using `Predict` in this function name. it is used only for spec-rec/other ML models. Rename it

Reply:
yes actually this function is not used in cropcert but i wanted this file as same as image.ts in biodiv ui , we can remove this part  .